docs/topics/migrations.txt

```diff
@@ -333,6 +336,12 @@ Note that this only works given two things:
   that your database doesn't match your models, you'll just get errors when
   migrations try to modify those tables.
 
+Note: if you have upgraded from Django 1.7 or before, and get
+'foreign key constraint' errors (for example, when you run manage.py test),
```

Comment:
But I do think the main documentation could do with a comment to highlight what you can't do; I'd made some migrations for some of my apps in my project but not others; basically only created them when I needed to make a change to the model. And I didn't see anywhere in the doco a rule that you couldn't do that.